builders/get-started/eth-compare/consensus-finality.md

```diff
@@ -39,7 +39,7 @@ In terms of finality, blocks on Ethereum can take quite a bit longer to finalize
 
 Moonbeam does not use checkpoint blocks and instead relies on Polkadot's GRANDPA finality gadget, where the finality process is completed in parallel to block production. In addition, the finality process incorporates the blockchain's structure, which allows the relay chain validators to vote on the highest block that they think is valid. In this scenario, the vote would apply to all of the blocks leading up to the one that is finalized, which speeds up the finalization process. After a block has been included in the relay chain, a block can be finalized within one block on Moonbeam.
 
-## Strategy to Check Transaction Finality {: #strategy-to-check-tx-finality }
+## Check Transaction Finality with Ethereum RPC Endpoints {: #strategy-to-check-tx-finality }
```

Comment:
Shouldn't we have the tags match exactly or pretty close to the title? e.g. #check-transaction-finality